### PR TITLE
fix newSdp = null in PeerConnectionObserver.java

### PR DESCRIPTION
**Pull request by qusaieilouti99**
and comment the logger

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ import RTCSessionDescription from './RTCSessionDescription';
 import RTCView from './RTCView';
 import ScreenCapturePickerView from './ScreenCapturePickerView';
 
-Logger.enable('*');
+// Logger.enable('*');
```
Comment (saghul) [inline src/index.ts:18]: Please leave this the way it was.

File: android/src/main/java/com/oney/WebRTCModule/PeerConnectionObserver.java [hunk 2]
```diff
@@ -396,8 +396,11 @@ public void onIceCandidate(final IceCandidate candidate) {
         params.putMap("candidate", candidateParams);
         SessionDescription newSdp = peerConnection.getLocalDescription();
         WritableMap newSdpMap = Arguments.createMap();
-        newSdpMap.putString("type", newSdp.type.canonicalForm());
-        newSdpMap.putString("sdp", newSdp.description);
+        // Can happen when doing a rollback.
+        if (newSdp != null) {
```
Comment (saghul) [inline android/src/main/java/com/oney/WebRTCModule/PeerConnectionObserver.java:400]: Don't we need to handle this in the JS layer?

**Review by saghul — COMMENTED** [review]
Can you please apply the same fix on iOS too? I left a couple of comments too, thank you!

File: android/src/main/java/com/oney/WebRTCModule/PeerConnectionObserver.java (hunk 2 above)
Comment (qusaieilouti99) [reply android/src/main/java/com/oney/WebRTCModule/PeerConnectionObserver.java:400]: It's already handled this way on peerConnectionSetLocalDescription on setSuccess in the native layer.
so I just copied that

Comment (qusaieilouti99) [reply android/src/main/java/com/oney/WebRTCModule/PeerConnectionObserver.java:400]: I'm not familiar with objective c, so if you're back can you please fix it on IOS.

Comment (saghul) [reply android/src/main/java/com/oney/WebRTCModule/PeerConnectionObserver.java:400]: Ok

**Pull request closed without merging**